x-pack/plugin/core/template-resources/src/main/resources/fleet-agents.json

```diff
@@ -270,6 +270,9 @@
                 "download_percent": {
                   "type": "double"
                 },
+                "download_rate": {
+                  "type": "double"
+                },
```

Comment:
We also added two more download fields `retry_error_msg` and `retry_until`, let's map those while we are in here:

They came from https://github.com/elastic/elastic-agent/pull/3845

Reply:
Relates https://github.com/elastic/kibana/issues/173370

Reply:
added here: https://github.com/elastic/elasticsearch/pull/103566/commits/9f2307516338d2199e2947b39a89563d28de7a0b

Reply:
Thanks!